app/parsers/bulkrax/application_parser.rb

```diff
@@ -262,10 +262,14 @@ def unzip(file_to_unzip)
     end
 
     def zip
-      FileUtils.rm_rf(exporter_export_zip_path)
-      Zip::File.open(exporter_export_zip_path, create: true) do |zip_file|
-        Dir["#{exporter_export_path}/**/**"].each do |file|
-          zip_file.add(file.sub("#{exporter_export_path}/", ''), file)
+      FileUtils.mkdir_p(exporter_export_zip_path)
```

Comment:
refer to the pr comment under "demo" for what this new file structure looks like.